driver/src/it/scala/org/mongodb/scala/DocumentationTransactionsExampleSpec.scala

```diff
@@ -43,8 +43,8 @@ class DocumentationTransactionsExampleSpec extends RequiresMongoDBISpec {
     client.getDatabase("hr").createCollection("employees").execute()
     client.getDatabase("hr").createCollection("events").execute()
 
-    updateEmployeeInfoWithRetry(client).execute() should equal(Completed())
-    client.getDatabase("hr").drop().execute() should equal(Completed())
+    updateEmployeeInfoWithRetry(client).execute()
```

Comment:
`Completed()` has been removed and `Completed` is now just a type alias pointing to `Void`.

This means all execute wont return anything so `should equal(Completed())` can be removed

Reply:
Minor chance of a break here.

If the user relied on `onNext()` being called with a Completed Token then that would have to be updated to use `onComplete()` instead.  This change is akin to the change requested in JAVA-3398